testing/endtoend/evaluators/operations.go

```diff
@@ -34,7 +34,7 @@ var churnLimit = 4
 var depositValCount = e2e.DepositCount
 
 // Deposits should be processed in twice the length of the epochs per eth1 voting period.
-var depositsInBlockStart = types.Epoch(math.Floor(float64(params.E2ETestConfig().EpochsPerEth1VotingPeriod) * 2))
```

Comment:
this was doing some weird type conversion stuff that is totally unnecessary since `EpochsPerEth1VotingPeriod` is a uint64 :man_shrugging: